orchestrator-salt/src/main/resources/salt/salt/cloudera/agent/init.sls

```diff
@@ -11,6 +12,8 @@ install-cloudera-manager-agent:
     - unless:
       - rpm -q cloudera-manager-daemons cloudera-manager-agent
 
+{# Protect this via a grains[prewarmed] check}
+{# TODO Question: Can the prewarmed tag be set in the image itself?}
```

Comment:
I think we can add default grains during image creation time, saying it's a pre-warmed image.